AIOFabricator/AiOFab.cs

```diff
@@ -40,6 +42,7 @@ public AiOFab()
                    "Multi-fuction fabricator capable of synthesizing most blueprints.")
         {
             OnStartedPatching += LoadImageFiles;
+            OnFinishedPatching += LoadImageFiles;
```

Comment:
Do we really need to run the `LoadImageFiles` routine twice?  
Would it be enough to only run on the `OnFinishedPatching` event?